cmd/search/static-services-entries.json

```diff
@@ -2,8 +2,108 @@
   {
     "id": "quay",
     "links": [
-      "quay.quayOrganizations",
-      "quay.quayRepositories"
+      {
+        "custom": true,
+        "id": "QUAY.quayOrganizations",
+        "href": "/quay/organization",
+        "title": "Quay organization",
+        "description": "Group and manage repositories belonging to a particular team, group, or company.",
+        "alt_title": [
```

Comment:
Can't we attach these to the existing links? I don't think we have to create entirely new entries.